pom.xml

```diff
@@ -0,0 +1,163 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>uk.gov.companieshouse</groupId>
+	<artifactId>company-accounts-library</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	<name>company-accounts-library</name>
+
+	<properties>
+
+		<!-- Encoding -->
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<!-- Dependency Versions -->
+		<avro.version>1.8.1</avro.version>
+		<commons-codec-version>1.9</commons-codec-version>
+		<commons-io.version>2.4</commons-io.version>
+		<jacoco-maven-plugin.version>0.7.7.201606060606</jacoco-maven-plugin.version>
+		<junit.version>4.12</junit.version>
+		<maven-compiler-plugin.version>3.5.1</maven-compiler-plugin.version>
+		<snakeyaml.version>1.17</snakeyaml.version>
+		<structured-logging.version>1.2.0-rc1</structured-logging.version>
+
+		<!-- JDK -->
+		<source-level>1.8</source-level>
+		<target-level>1.8</target-level>
+
+		<!-- Repositories -->
+		<artifactoryBaseUrl>http://ch-jenkins-2k8.internal.ch:8081/artifactory</artifactoryBaseUrl>
```

Comment:
We've changed the way we approach this to avoid hard-coding URLs into our pom. If you remove this, when you actually specify the repository URL (in the `<repository>` section, you can reference the maven repository url instead (`${MAVEN_REPOSITORY_URL}`). This *should* work, but give it a test. If we can avoid hard-coding repositories then great

Reply:
resolved in ecc5caa